wordpress-mobile/WordPress-Android
# Update FluxC - fix visibility of created sites

This PR fixes an issue with the "Create new site" endpoint. We were sending `"public": "PUBLIC"` parameter instead of `"public":"1"`. 

Corresponding PR in FluxC is [here](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1727).

More info p4a5px-2DW-p2

Merge instructions:
1. Create a new tag in Fluxc and update it 
2. Merge this PR

To test:
1. Open a network sniffer (eg. Stetho)
2. Click on My Site -> switch site -> Plus icon -> create .com site
3. Walk through the Site Creation flow
4. Click on "Create Site" button
5. Look at Stetho and check that request to /new endpoint contains `"public":"1"`
6. If the server side patch hasn't been applied yet, you can also go to site settings and check that the site visibility is `public`

PR submission checklist:
